beacons-data/export_modern_beacons.sql

```diff
@@ -0,0 +1,16 @@
+SELECT beacon.hex_id, 
+    UPPER(beacon.beacon_status) as beacon_status, 
+    TO_CHAR(beacon.created_date :: DATE, 'dd/mm/yyyy') as created_date,
```

Comment:
It's probably worth having the same column order, and same number of columns in both to make splicing easier for Linda.